db/migrate/20151202162648_create_payment_braintree_subscriptions.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePaymentBraintreeSubscriptions < ActiveRecord::Migration
+  def change
+    create_table :payment_braintree_subscriptions do |t|
+      t.string :subscription_id
+      t.datetime :next_billing_date
```

Comment:
If the next billing date only applies for the initial billing date and never gets updated, this column might be named in a little confusing manner. If that's the case and this is something we'll be using for our own use case rather than for just building the initial Braintree transaction, I'd change this to `first_billing_date`.